pagopa/io-app
# chore(IT Wallet): [SIW-1639] Show error banner when wallet instance status check fails

## Short description
This PR handles unexpected failures during the request to fetch the Wallet Instance status. When it is not possible to determine the status, the Wallet will not be available.

## List of changes proposed in this pull request
- Created `ItwWalletNotAvailableBanner` component
- Turned `itWallet.walletInstance.status` into a pot for better async handling (+ store migration)
- Hide `ItwWalletReadyBanner` and `ItwWalletCardsContainer` when the Wallet Instance status call failed

## How to test
- Get a wallet instance, then fake an error in `api/v1/wallet/wallet-instances/<wallet_instance_id>/status`: you should see the following screen

Regression tests:
- Get a wallet instance and restart the app: everything should work as before
- Get a wallet instance, then revoke it: everything should work as before (with an alert only shown once)

<img src="https://github.com/user-attachments/assets/cf8d36d0-0fad-4e5e-a4a8-80b75ce3c72b" width="240" />
